stdlib/src/random/random.mojo

```diff
@@ -201,3 +201,28 @@ fn randn[
     for i in range(size):
         ptr[i] = randn_float64(mean, variance).cast[type]()
     return
+
+
+fn shuffle[T: CollectionElement](inout list_in: List[T]):
```

Comment:
```suggestion
fn shuffle[T: CollectionElement, //](inout list: List[T]):
```

Reply:
Why inferred here?

Reply:
IIUC, this is the direction the stdlib is moving towards. We won't need to manually specify this type parameter anyways.